### DISCUSSION TITLE
Backup pi4 to Google cloud

### DISCUSSION THREAD

**Discussion by pocoyoman** [teslamate-org/teslamate#3253] [General]
Hello 
I made a backup (teslamate.bck) of my PI4
Now I have a functional google cloud instance for teslamate and I would like to restore the pi4 backup to google cloud but I can't! (I was able to recover the file from the pi4) but later I get stuck...)

**Comment by ktsenvolt** [comment]
What was the error message?

**Reply by pocoyoman** [reply]
It's ok now

**Comment by vzhivkov** [comment]
> Hello I made a backup (teslamate.bck) of my PI4 Now I have a functional google cloud instance for teslamate and I would like to restore the pi4 backup to google cloud but I can't! (I was able to recover the file from the pi4) but later I get stuck...)

Hello, how did you made that google cloud instance running? Can you give me a link to a working tutorial or something, please?

**Reply by ktsenvolt** [reply]
https://www.teslaev.co.uk/how-to-perform-an-automatic-teslamate-backup-to-google-drive/
